### PR TITLE
Add Documentation to `.to_edge` and `to_corner`

### PR DESCRIPTION
**Pull request by TheMathematicFanatic**
## Overview: What does this pull request change?

Added documentation and example renders for mobject's .to_edge and .to_corner methods

https://manimce--3408.org.readthedocs.build/en/3408/reference/manim.mobject.mobject.Mobject.html?highlight=to_edge#manim.mobject.mobject.Mobject.to_edge
https://manimce--3408.org.readthedocs.build/en/3408/reference/manim.mobject.mobject.Mobject.html?highlight=to_corner#manim.mobject.mobject.Mobject.to_corner

### CODE REVIEW

**Review by Uiblar — APPROVED** [review]
Simple and clear/understandable. Thanks!

**Review by Uiblar — APPROVED** [review]

**Review by behackl — CHANGES_REQUESTED** [review]
Thanks for your contribution! Two minor issues with your current proposal:

- the added docstrings are missing the `Parameters`-section which explain the input arguments,
- the two examples you chose are nice in principle -- however, they don't strictly need to be animated. (We are currently running into an issue with our documentation build time, and every added rendered animation tends to add a few more seconds to it -- static images are more or less fine though.) Could you think of a way where these two methods are illustrated with static images instead?

File: manim/mobject/mobject.py [hunk 1]
```diff
@@ -1407,9 +1407,62 @@ def align_on_border(self, direction, buff=DEFAULT_MOBJECT_TO_EDGE_BUFFER):
...
+                    c.to_corner(UR)
+                    t = Tex("To the corner!")
+                    t2 = MathTex("x^3").shift(DOWN)
+                    self.add(c,t,t2)
+                    self.wait()
+                    self.play(
+                        t.animate.to_corner(DL, buff=0),
+                        t2.animate.to_corner(UL, buff=1.5)
+                    )
+                    self.wait()
```
Comment (MrDiver) [inline manim/mobject/mobject.py:1434]:
```suggestion
        .. manim:: ToCornerExample

            class ToCornerExample(Scene):
                def construct(self):
                    c = Circle()
                    c.to_corner(UR)
                    t = Tex("To the corner!")
                    t2 = MathTex("x^3").shift(DOWN)
                    self.add(c,t,t2)
                    t.to_corner(DL, buff=0)
                    t2.to_corner(UL, buff=1.5)
```

File: manim/mobject/mobject.py [hunk 2]
```diff
@@ -1407,9 +1407,62 @@ def align_on_border(self, direction, buff=DEFAULT_MOBJECT_TO_EDGE_BUFFER):
...
+                    tex_top.to_edge(UP)
+                    tex_side = Tex("I am moving to the side!")
+                    c = Circle().shift(2*DOWN)
+                    self.add(tex_top, tex_side)
+                    self.wait()
+                    self.play(
+                        tex_side.animate.to_edge(LEFT),
+                        c.animate.to_edge(RIGHT, buff=0)
+                    )
+                    self.wait()
```
Comment (MrDiver) [inline manim/mobject/mobject.py:1464]:
```suggestion
            class ToEdgeExample(Scene):
                def construct(self):
                    tex_top = Tex("I am at the top!")
                    tex_top.to_edge(UP)
                    tex_side = Tex("I am moving to the side!")
                    c = Circle().shift(2*DOWN)
                    self.add(tex_top, tex_side)
                    tex_side.to_edge(LEFT)
                    c.to_edge(RIGHT, buff=0)
                    
```

**Review by MrDiver — COMMENTED** [review]
Changes for the animations to be static

File: manim/mobject/mobject.py [hunk 3]
```diff
@@ -1407,9 +1407,55 @@ def align_on_border(self, direction, buff=DEFAULT_MOBJECT_TO_EDGE_BUFFER):
...
+        -------
+        :class:`.Mobject`
+            The newly positioned mobject.
+
+        Examples
+        --------
+
+        .. manim:: ToCornerExample
+
+            class ToCornerExample(Scene):
```
Comment (MrDiver) [inline manim/mobject/mobject.py:1422]:
```suggestion
        .. manim:: ToCornerExample
            :save_last_frame:

            class ToCornerExample(Scene):
```

File: manim/mobject/mobject.py [hunk 4]
```diff
@@ -1407,9 +1407,55 @@ def align_on_border(self, direction, buff=DEFAULT_MOBJECT_TO_EDGE_BUFFER):
...
+        -------
+        :class:`.Mobject`
+            The newly positioned mobject.
+
+        Examples
+        --------
+
+        .. manim:: ToEdgeExample
+
+            class ToEdgeExample(Scene):
```
Comment (MrDiver) [inline manim/mobject/mobject.py:1448]:
```suggestion
        .. manim:: ToEdgeExample
            :save_last_frame:
            class ToEdgeExample(Scene):
```

File: manim/mobject/mobject.py [hunk 5]
```diff
@@ -1423,11 +1423,58 @@ def align_on_border(
...
+        -------
+        :class:`.Mobject`
+            The newly positioned mobject.
+
+        Examples
+        --------
+
+        .. manim:: ToEdgeExample
+            :save_last_frame:
+            class ToEdgeExample(Scene):
```
Comment (MrDiver) [inline manim/mobject/mobject.py:1467]:
```suggestion
            :save_last_frame:
            
            class ToEdgeExample(Scene):
```

**Review by JasonGrace2282 — APPROVED** [review]
LGTM

**Pull request merged**